thoth/storages/graph/postgres.py

```diff
@@ -3606,6 +3607,54 @@ def _runtime_environment_conf2models(
 
         return hardware_information, software_environment
 
+    def create_github_app_installation(self, install_data: List[dict]) -> int:
+        """Create a record for new installation.
+
+        Example -
+        install_data =  [
+            {
+                "id": "236796147",
+                "node_id": "MDEwOlJlcG9zaXRvcnkyMzY3OTYxNDc=",
+                "name": "advisor",
+                "full_name": "thoth-station/advisor",
+                "private": false
+            }
+        ]
+        Accepts a list of installation details passed down by Github.
+        Returns a count of items added.
+        """
+        count = 0
+        with self._session_scope() as session:
+            for repository in install_data:
+                _, already_exist = KebechetGithubAppInstalltions.get_or_create(
+                    session,
+                    slug=repository.get("full_name"),
+                    repo_name=repository.get("name"),
+                    private=repository.get("private"),
+                    installation_id=repository.get("id"),
+                )
+                count += 1 if not already_exist else 0
+            return count
+
+    def delete_github_app_installation(self, uninstall_data: List[dict]) -> int:
```

Comment:
What about introducing a flag "`is_active`" here so we deactivate/activate the installation?

Reply:
Okay, so the logic would be to keep them in the table after they uninstall and just deactivate them.

Reply:
Should I alter the uninstall method? or change add a method to add to deactivate?

Reply:
Check already existing queries - I think most of them use `update_`.

BTW @pacospace I think we are missing this scenario in the naming convention?

Reply:
Let's add it if missing thanks :)

Reply:
@pacospace sorry I didn't get what do we need?

Reply:
https://github.com/thoth-station/storages/blob/master/docs/conventions/README.md conventions for naming of the query, if you think you can reuse what we have or we need a new type add in the PR :) @saisankargochhayat

Reply:
@pacospace Thanks, I have added it in https://github.com/thoth-station/storages/pull/1768/commits/e646c78aebae9ecdf26db3c2b8a7d520dc6aea60